api-tests/.env.local

```diff
@@ -6,7 +6,7 @@ CORE_BACK_EXTERNAL_API_URL="http://localhost:4502"
 CORE_BACK_PUBLIC_ENCRYPTION_KEY='{"kty": "RSA","n": "0465qJwo8nCkC2tvV4niuWF6IM6pNjmeYszhTwHPY609-HVAtO8PoRLUyA86rzQ-QzbT7XxbzCjfyRXoRFOGleZqTuwlc25ezDxV58bhecPiWFMaFYOS1W7zIDsVFo37gjjvtkcD6OqK8PKAv6n5tUphjDCcnnmpTMIyGAnzmQCbSkJWu6V_gc3tirAugXoZukMCohxw3_-c6prhMN0smDNv0qWmva3oqokabePwe1OS72DXyXR-TPd_Dtz4-tRr9jvZwHulX4Zcs1BBbjBpIim3WNY8asv9yjlBxkdt-nckhCMZekPuT7xWSTrvccB_fnnSUgEQW_5irLNdnr5MWQ","e": "AQAB","alg": "RS256"}' # pragma: allowlist secret
```

Comment:
Do the encryption keys need a `kid` as well?

Reply:
I suppose the alternative is to read the encryption key from the Core JWKS endpoint instead 🤔

What do you think? Should be a separate PR, but we have the method in `core-back-external-client` already! (Should probably only fetch once per test run though)

Reply:
that's a good idea, I'll do that as part of a separate PR